README.md

```diff
@@ -123,6 +123,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 | <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/drupal.svg#gh-light-mode-only" alt="Drupal" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/drupal-white.svg#gh-dark-mode-only" alt="Drupal" align=left width=24 height=24> [Drupal module](https://www.drupal.org/project/simple_icons) | [Phil Wolstenholme](https://www.drupal.org/u/phil-wolstenholme) |
 | <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/figma.svg#gh-light-mode-only" alt="Figma" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/figma-white.svg#gh-dark-mode-only" alt="Figma" align=left width=24 height=24> [Figma plugin](https://www.figma.com/community/plugin/1149614463603005908/Simple-Icons) | [@LitoMore](https://github.com/LitoMore) |
 | <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/flutter.svg#gh-light-mode-only" alt="Flutter" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/flutter-white.svg#gh-dark-mode-only" alt="Flutter" align=left width=24 height=24> [Flutter package](https://pub.dev/packages/simple_icons) | [@jlnrrg](https://jlnrrg.github.io/) |
+| <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/go.svg#gh-light-mode-only" alt="Go" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/go-white.svg#gh-dark-mode-only" alt="Go" align=left width=24 height=24> [Golang plugin WIP](https://github.com/marcopollivier/go-simple-icons/) | [@marcopollivier](https://github.com/marcopollivier) |
```

Comment:
The `go-white.svg` needs to be added.

You need to copy-paste one from `icons/go.svg` and add `fill="white"` to `go-white.svg`

Reply:
```suggestion
| <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/go.svg#gh-light-mode-only" alt="Go" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/go-white.svg#gh-dark-mode-only" alt="Go" align=left width=24 height=24> [Golang plugin](https://github.com/marcopollivier/go-simple-icons/) | [@marcopollivier](https://github.com/marcopollivier) |
```